components/misc/rng.cpp

```diff
@@ -1,28 +1,24 @@
 #include "rng.hpp"
 #include <cstdlib>
-#include <ctime>
 
 namespace Misc
 {
 
-    void Rng::init()
-    {
-        std::srand(static_cast<unsigned int>(std::time(NULL)));
-    }
-
     float Rng::rollProbability()
     {
-        return static_cast<float>(std::rand() / (static_cast<double>(RAND_MAX)+1.0));
+        Rng generator;
```

Comment:
Every call for this method will produce initialization for random_device. It is will cause on perfomance.